examples/widgets/label/lv_example_label_1.c

```diff
@@ -14,6 +14,7 @@ void lv_example_label_1(void)
     lv_obj_set_width(label1, 150);  /*Set smaller width to make the lines wrap*/
     lv_obj_set_style_text_align(label1, LV_TEXT_ALIGN_CENTER, 0);
     lv_obj_align(label1, LV_ALIGN_CENTER, 0, -40);
+    lv_obj_set_style_text_font(label1, &lv_font_montserrat_30, 0);
```

Comment:
Probably it was, but could you revert it?

Reply:
It was added by you in the closed PR [here](https://github.com/kisvegabor/lvgl_fork/commit/ee3229870cb50c3bb5d5ed6c52f95671f775e578#diff-e96f9ac60505e20c53945a5ee1bd326fd627943465067fd10d25ff74ed374466R17). Do you want me to revert to use the default font, or to a specific font?

Reply:
Sorry I meant, "Probably it was **me**, but could you revert it?" 

So, yes, please remove it. I added it only for testing.